core/templates/dev/head/components/top_navigation_bar/TopNavigationBarDirective.js

```diff
@@ -80,7 +80,7 @@ oppia.directive('topNavigationBar', [
           var NAV_MODES_WITH_CUSTOM_LOCAL_NAV = [
             'create', 'explore', 'collection', 'topics_and_skills_dashboard',
             'topic_editor', 'story_editor'];
-          $scope.currentUrl = window.location.pathname.split("/")[1];
+          $scope.currentUrl = window.location.pathname.split('/')[1];
```

Comment:
why this change? I think if you merge develop with this branch, all linting errors should be fixed.

Reply:
its someone error , when i update the local repo then someones issue is there.